app/models/sdg/target.rb

```diff
@@ -0,0 +1,16 @@
+class SDG::Target < ApplicationRecord
+  validates :code, presence: true, uniqueness: { scope: :goal_id }
```

Comment:
I don't have a strong preference about scoping the unique index. However, I think we should do the same thing in both the model and the database; right now the model is scoped but the database index is not :thinking:.

Reply:
As the first part of the code of any target is the code of the parent goal, maybe we can rely on the current unique index on code and do not use the scope. 🤔

Reply:
Sounds good to me :+1:.